iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/checks/KubernetesVerifier.java

```diff
@@ -239,8 +241,13 @@ private static HelmInputFileContext prepareHelmContext(String templateFileName)
           filePath));
       }
       addDependentFilesToSensorContext(helmProjectPath);
+      var helmInputFileContext = new HelmInputFileContext(SENSOR_CONTEXT, sourceInputFile);
+      var additionalFiles = StreamSupport.stream(SENSOR_CONTEXT.fileSystem().inputFiles(f -> true).spliterator(), false)
+        .collect(Collectors.toMap(file -> file.uri().toString(), file -> file));
 
-      return new HelmInputFileContext(SENSOR_CONTEXT, sourceInputFile);
+      helmInputFileContext.setAdditionalFiles(additionalFiles);
```

Comment:
In #1383, I solved this slightly differently: these files will be set during parsing, so we need to create ProjectContext after parsing to make everything work

Reply:
Ok, so please remove it in your PR.